tests/ui/test_discopane.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from pages.discopane import DiscoveryPane
+
+@pytest.fixture(scope='session')
+def base_url(base_url):
+    # This is hardcoded for now, once we go through about:addons, we'll be able to use the template expansion
+    if base_url[len(base_url)-1] != '/':
+        base_url += '/'
+    base_url += 'en-US/firefox/discovery/pane/48.0/Darwin/normal'
+    if 'http' not in base_url:
+        base_url = 'https://' + base_url
```

Comment:
This would prevent us from testing against sites hosted without SSL, which could be intentional. Let's leave it to the user to specify the correct base URL.